lib/menu.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'dish'
+
+class Menu
+  attr_accessor :dishes
+
+  def initialize
+    @dishes = []
+  end
+
+  def add_dish(dish) 
+    # dish is instance of dish class
+    # raise "add_dish expects an instance of Dish class" unless dish.is_a?(Dish)
+    @dishes.push(dish)
+  end
+
+  def display
+    @dishes
+  end
+
```

Comment:
Does `#dish?` check that a specific dish exists on the menu?  If so, you'll probably find that there's a simpler way to do this, with just one `Array` method ;)

Reply:
Also, perhaps an alternative name would make your intent clearer.  How about something like `#valid_dish?` or `#dish_present?` As you can see, naming is hard, but putting some time into getting it right is worthwhile.